tools/cooja/build.gradle

```diff
@@ -0,0 +1,88 @@
+allprojects {
+  apply plugin: 'java'
+
+    repositories {
+      jcenter()
+    }
+
+}
+
+apply plugin: 'findbugs'
+apply plugin: 'checkstyle'
+apply plugin: 'application'
+apply plugin: 'eclipse'
+apply plugin: 'idea'
```

Comment:
I'm less concerned about fixing, but adding new ones. Maybe a comment in the code would be nice.